docs/src/prototypes/chatMessages/ChatMessageWithPopover/Popover.tsx

```diff
@@ -4,6 +4,8 @@ import cx from 'classnames'
 
 export interface PopoverProps {
   className?: string
+  setFixedMode?: React.Dispatch<React.SetStateAction<boolean>>
```

Comment:
Not sure I understand the requested changes. I can rename these to `setOpen` and `setForceOpen` or `forceOpen`, but those are callback, not sure how to change them to booleans...

Reply:
yeah, I see - in that case, still, by following Controlled pattern, lets name props accordingly: they are designed to request state changes, so lets use `on...` naming convention, something like the following. This will be just about rename refactoring.

```jsx
...
 return (
    <Chat.Message
       ...
       actionMenu={<Popover onForceOpenChange={setFixedMode} onOpenChange={setOpen} />}
/>)
```

Reply:
I renamed that using the state names, `showActionMenu` => `onShowActionMenuChange` and `forceShowActionMenu` => `onForceShowActionMenuChange`